dag/main.yml

```diff
@@ -344,8 +344,36 @@ steps:
   # Happiness report
   data://meadow/happiness/2023-03-20/happiness:
   - snapshot://happiness/2023-03-20/happiness.xls
+  data://meadow/happiness/2022-03-20/happiness:
+  - snapshot://happiness/2022-03-20/happiness.xls
+  data://meadow/happiness/2021-03-20/happiness:
+  - snapshot://happiness/2021-03-20/happiness.xls
+  data://meadow/happiness/2020-03-20/happiness:
+  - snapshot://happiness/2020-03-20/happiness.xls
+  data://meadow/happiness/2019-03-20/happiness:
+  - snapshot://happiness/2019-03-20/happiness.xls
+  data://meadow/happiness/2018-03-20/happiness:
+  - snapshot://happiness/2018-03-20/happiness.xls
+  data://meadow/happiness/2017-03-20/happiness:
+  - snapshot://happiness/2017-03-20/happiness.xlsx
+  data://meadow/happiness/2016-03-20/happiness:
+  - snapshot://happiness/2016-03-20/happiness.xlsx
+  data://meadow/happiness/2015-03-20/happiness:
+  - snapshot://happiness/2015-03-20/happiness.xlsx
+  data://meadow/happiness/2012-03-20/happiness:
+  - snapshot://happiness/2012-03-20/happiness.xlsx
+
   data://garden/happiness/2023-03-20/happiness:
   - data://meadow/happiness/2023-03-20/happiness
+  - data://meadow/happiness/2022-03-20/happiness
+  - data://meadow/happiness/2021-03-20/happiness
+  - data://meadow/happiness/2020-03-20/happiness
+  - data://meadow/happiness/2019-03-20/happiness
+  - data://meadow/happiness/2018-03-20/happiness
+  - data://meadow/happiness/2017-03-20/happiness
+  - data://meadow/happiness/2016-03-20/happiness
+  - data://meadow/happiness/2015-03-20/happiness
+  - data://meadow/happiness/2012-03-20/happiness
   - backport://backport/owid/latest/dataset_235_world_happiness_report__2022
```

Comment:
is the `backport` dataset a dependency still? I don't see any reference to it in the code.